.github/workflows/mergeabot.yml

```
@@ -13,6 +13,13 @@ jobs:
   mergeabot:
     runs-on: ubuntu-latest
     steps:
+      - id: token
```

Might be worth a comment,

```suggestion
      # Use an app token so further workflows (e.g. release) are triggered
      - id: token
```